### DISCUSSION TITLE
Difficoltà nell'integrare impianto Aqara in Home Assistant

### DISCUSSION THREAD

**Discussion by officialdavide** [indomus/forum#2577] [HUB: Home Assistant]
Ciao ragazzi e ragazze,
chiedo supporto a Voi in quanto sono pieno di dubbi e non trovo via di uscita.

**Preambolo:** ho installato Home Assistant Core (con Docker) e poi ho un "impianto Aqara" (hub M1S, più due coppie di sensori porte e rilevatore temperatura/pressione/umidità)

L'impianto Aqara ormai funziona benissimo e l'avevo configurato con il mio telefono Android.
Non ho mai associato alcun di questi componenti ad _Apple HomeKit_ (mia moglie ha un telefono Apple ma non ha mai voluto collegarli).

Ieri ho installato Home Assistant Core (con Docker) come da vostra guida e mi aspettavo di vedere in _Impostazioni_ > _Integrazioni_ (almeno) l'hub M1S di Aqara.

Allora, tramite il pulsante _Aggiungi integrazione_, ho aggiunto _HomeKit Bridge_ e mi ha creato l'integrazione. 
Ma ora come procedo? Avete qualche guida (sicuramente mi sarà sfuggita - il bello di inDomus.it che ce ne sono parecchie! 👍 ).

Grazie mille!

**Comment by KC-inDomus** [comment]
Se vuoi usare l'integrazione HomeKit Device devi rimuovere il bridge da dentro l'app Casa di iOS (se stai usando iPhone)

**Reply by officialdavide** [reply]
Ciao, grazie della risposta. 

Nel mio post non ho mai collegato il bridge all'app "Casa" di iOS. Ho solo collegato l'hub all'app Aqara con un telefono Android. 

Mi piacerebbe, almeno all'inizio, poter accendere l'hub di Aqara con colori diversi. 

Posso chiederti, per favore, quali passi dovrei fare una volta che ho fatto accesso alla parte web di Home Assistant? (o dove posso trovare una guida specifica)
